### ISSUE TITLE
A folder with an unspecified path is created when user execute command `vatz init`

### ISSUE THREAD

**Issue by xellos00** [dsrvlabs/vatz#458]
### Checklist
- [x] Bug from Service (Vatz)
- [ ] Bug from Plugin (Vatz) 
- [ ] Malfunction on CI/CD 
- [ ] etc(e.g, Hardware version doesn't match, )

---

### **Describe the bug**
> A clear and concise description of what the bug is.

Even if we 

### **How to Reproduce**
> Steps to reproduce the behavior:

1. Check the folder
<img width="412" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/4102aba4-4bec-4551-a0a2-28e7439556ce">

2. Execute `vatz init`

3. Check the folder there's `%s` folder. 

<img width="412" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/302cd631-7c8c-42de-a9ad-ced40e92b229">

### **Expected behavior**
> A clear and concise description of what you expected to happen.

I thought our default home was  "~/.vatz"

### **Screenshots**
> If applicable, add screenshots to help explain your problem.

**Comment by rootwarp** [comment]
@xellos00 
I inspected this problem and I found what the problem is.

#433 was merged to d4db181 and the build is broken from ea6eccf

When I get the diffs between ea6eccf and d4db181, critical part was removed from the `init.go`.

Below is the content of the diff command.
```
rootwarp@mindhack ~/g/d/vatz ea6eccf? > git --no-pager diff 3cf7866
...
...
...
-                       homePath, err := cmd.Flags().GetString("home")
-                       if err != nil {
-                               return err
-                       }
-
-                       template = fmt.Sprintf(template, homePath)
```

Due to `template = fmt.Sprintf(template, homePath)` part is removed the format string is not substituted to real home path.

**Comment by xellos00** [comment]
> @xellos00 I inspected this problem and I found what the problem is.
> 
> #433 was merged to [d4db181](https://github.com/dsrvlabs/vatz/commit/d4db181076cc615785291fed248bfe9c0ba5bc4d) and the build is broken from [ea6eccf](https://github.com/dsrvlabs/vatz/commit/ea6eccfaa2bee9131aef50d5c052e334780cf1cb)
> 
> When I get the diffs between [ea6eccf](https://github.com/dsrvlabs/vatz/commit/ea6eccfaa2bee9131aef50d5c052e334780cf1cb) and [d4db181](https://github.com/dsrvlabs/vatz/commit/d4db181076cc615785291fed248bfe9c0ba5bc4d), critical part was removed from the `init.go`.
> 
> Below is the content of the diff command.
> 
> ```
> rootwarp@mindhack ~/g/d/vatz ea6eccf? > git --no-pager diff 3cf7866
> ...
> ...
> ...
> -                       homePath, err := cmd.Flags().GetString("home")
> -                       if err != nil {
> -                               return err
> -                       }
> -
> -                       template = fmt.Sprintf(template, homePath)
> ```
> 
> Due to `template = fmt.Sprintf(template, homePath)` part is removed the format string is not substituted to real home path.

got it. I wii put this back.
